managedtenants/utils/ocm.py

```diff
@@ -335,15 +336,18 @@ def _addon_from_imageset(self, imageset, metadata):
                     continue
                 if key == "config":
                     addon = self.set_addon_config(
-                        addon=addon, config_obj=val, mapped_key=mapped_key
+                        addon=addon,
+                        metadata=metadata,
```

Comment:
Would it be easier to just pass the addonID around? wdyt?

Reply:
fair enough, updating :)